### ISSUE TITLE
open plugin project in xcode (error: no such module capacitor)

### ISSUE THREAD

**Issue by gcaqp** [ionic-team/capacitor#949]
Hi,
I created a plugin with the capacitor generator
npx @capacitor/cli plugin:generate

and execute the following commands
-android
-ios
   -Plugin
   -Plugin.xcodeproj
   -Plugin.xcworkspace
   -Plugin
   -Podfile

and execute the following commands
npm install
npm build
npx cap sync
        Error: Capacitor... could not find  the web aseets directory /www
npx cap open ios
       Error: Xcode workspace does not exist ...  

I need to open the project in xcode to debugger and develop some unit tests, I opened the project manually but it shows a message "No such module capacitor"

thx

**Comment by jcesarmobile** [comment]
Can you rephrase your questions? it's not clear what you are saying

**Comment by gcaqp** [comment]
it's simple, I want to know how to open with xcode a plugin project, generated with "npx @capacitor/cli plugin:generate" without showing the error  "No such module capacitor" 

code:
_import Capacitor_ (here show error "No such module Capacitor")

**Comment by jcesarmobile** [comment]
yeah, but what's not clear is your steps. 
In example

> and execute the following commands
> -android
> -ios
> -Plugin
> -Plugin.xcodeproj
> -Plugin.xcworkspace
> -Plugin
> -Podfile

that whole part makes no sense to me

And the other part is not very clear neither, if you are in a plugin folder you can't run npx cap sync because it's not a Capacitor project, but a plugin folder.

So, can you rephrase the whole question, making it clear steps and commands and from where you ran them?

**Comment by mlynch** [comment]
Also says the Xcode workspace does not exist. Something isn’t right

**Comment by jcesarmobile** [comment]
Closing as the requested information was not provided and it's not clear what the problem is.

The plugin folder should work if you run `pod install` in the generated plugin and then open the .xcworkspace instead of the .xcodeproj

**Comment by digaus** [comment]
> Closing as the requested information was not provided and it's not clear what the problem is.
> 
> The plugin folder should work if you run `pod install` in the generated plugin and then open the .xcworkspace instead of the .xcodeproj

Then you should update your docs... nowhere does it state you need to run pod install

**Comment by stevesobol** [comment]
Seconded.  I didn't know to run `pod install` either.

https://capacitor.ionicframework.com/docs/plugins/ios/ does not say we need to run `pod install`.

Do you guys do pull requests for documentation? This is a problem I can fix easily.

**Comment by jcesarmobile** [comment]
The plugin generation runs pod install now
https://github.com/ionic-team/capacitor/blob/master/cli/src/tasks/new-plugin.ts#L134
